### DISCUSSION TITLE
Reverse sort order for seasons only

### DISCUSSION THREAD

**Discussion by ManCaveMedia** [bootsie123/F1-Web-Viewer#27] [Q&A] [answered]
Last race I noticed the Season dropdown is sorted in reversed sort order, while all others are not.
To select the live F1 race (or practice/qualification) feed you leave Series to Formula 1, select the top option for Season (2021) and the bottom option for Events and Session.

I'm curious what everyon thinks would be the most logical sort order? Latest at the bottom or at the top for all dropdowns?

**Comment by bootsie123** [comment, marked as answer]
Hi! Sorry I took me a few days to respond. Having the ordering be consistent is definitely a good idea. I think it makes most sense to have the latest be on top. Although that's just me.

I'd also be curious what everyone's thoughts are!

**Reply by ManCaveMedia** [reply]
Yeah, latest on top makes the most sense to me too.
Makes it easier to get to the live event or latest event.

**Reply by bootsie123** [reply]
I know it's been ages since we discussed this, but for anyone looking at this in the future, most recent/latest event is currently implemented on a consistent basis
